src/utils/createMedicJson.js

```diff
@@ -0,0 +1,16 @@
+// /usr/bin/env node
+/*jshint node: true*/
+
+var fs           = require('fs'),
+    path         = require('path'),
+    error_writer = require('../build/makers/error_writer');
+
+module.exports = function createMedicJson(output, sha, cfg) {
+    try {
+        console.log('Writing medic.json to ' + output);
+        var medic_config = '{"sha":"' + sha + '","couchdb":"' + cfg.couchdb.host + '","couchdbext":"' + cfg.couchdb.exthost + '"}';
+        fs.writeFileSync(path.join(output, 'medic.json'), medic_config, 'utf-8');
+    } catch (e) {
```

Comment:
Why are we swallowing this exception and continuing - shouldn't we not continue if this step fails.